locales/en/chef-profile.json

```
@@ -0,0 +1,10 @@
+{
+    "add-gear": "add gear",
+    "additional-person": "For each additional person",
+    "base-price": "base price",
+    "children-discount": "Would you like to offer a child discount?",
+    "max-count-persons": "(Max. number of people {{count}} )",
```

I think this white space is too much. If so, it would have to be removed from all translations

```suggestion
    "max-count-persons": "(Max. number of people {{count}})",
```